packages/react/src/components/Menu/Menu.tsx

```diff
@@ -23,7 +24,7 @@ import MenuDivider from './MenuDivider'
 
 export type MenuShorthandKinds = 'divider' | 'item'
 
-export interface MenuProps extends UIComponentProps, ChildrenComponentProps {
+export interface MenuProps extends UIComponentProps, ChildrenComponentProps, ColorComponentProps {
```

Comment:
Added color prop to all Menu* components, as a showcase of how the color prop should be implemented.